src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -23,6 +24,7 @@ export const PartyMap: React.FC = () => {
   useConnectCurrentVenue();
   const { user, profile } = useUser();
   const { recentVenueUsers } = useRecentVenueUsers();
+  const { isLoaded } = useCustomSoundsContext();
```

Comment:
To make this variable more meaningful in this context:

```suggestion
  const { isLoaded: isCustomSoundsLoaded } = useCustomSoundsContext();
```

Though it raises the question, do we want to block the entire venue from being accessible just because the sounds aren't loaded yet? I feel like maybe we shouldn't be doing that, as it'll still be largely usable even before the sounds exist.

By checking `isLoaded` at various points in the `useCustomSounds` hook, we can suppress any of the Bugsnag error reporting that are false positives because the data hasn't loaded yet. I expect that will make the need to use this here irrelevant.

So we can probably just remove it outright:

```suggestion
```

We also probably don't want to be using the context directly in components like this. It's more of an 'implementation detail' of the 'private API' that the `useCustomSounds` hook uses.

Reply:
got it, removed from `useCustomSoundsContext` and used `user` from `src/hooks/useUser.ts`, we shouldn't block the main page if the sounds is loading.